### DISCUSSION TITLE
"Couldn't load plugin 'UMEP-SuPy-QGIS3' due to an error when calling its classFactory() method" Error

### DISCUSSION THREAD

**Discussion by efdalkaya** [UMEP-dev/UMEP#631] [General]
Hi all. I am using QGIS 3.34.8. After the download UMEP Plugin zip file in github, set up from zip file section in QGIS than get a error message "Couldn't load plugin 'UMEP-SuPy-QGIS3' due to an error when calling its classFactory() method". how can ı solve problem?

**Comment by biglimp** [comment]
Can you give us more information on your error. There should be more debugging info also, what OS are you using?

**Reply by efdalkaya** [reply]
OS is win 11. The error below

![1](https://github.com/UMEP-dev/UMEP/assets/17739723/1b25aa87-e64c-4196-9796-395309be6933)

**Reply by j3r3m1** [reply]
What is your pandas version ? To know that, in a Python console in QGIS, you can type the following line by line:
```python
import pandas
print(pandas.__version__)
```

**Reply by efdalkaya** [reply]
I solved problem to set up old pandas version now run qgis :)
